app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -49,7 +49,11 @@ module ContentFacetHostExtensions
         )
 
         def content_facet_ignore_update?(attributes)
-          self.content_facet.blank? && attributes.values.all?(&:blank?)
+          self.content_facet.blank? && (
+            attributes.values.all?(&:blank?) ||
+            attributes['content_view_id'].blank? ||
+            attributes['lifecycle_environment_id'].blank?
```

Comment:
Oh yeah, I will update this and also add host update tests.

Reply:
I'm still testing this, but I think it's actually fine as is. This only affects creation of the content_facet because we're checking `self.content_facet.blank?`. Either way, I'm going to write a test for the update case.

Reply:
Though, the method name is pretty misleading if that's correct because there is no ignoring a content_facet update, only content_facet create. :P

Reply:
oh yeah, good point!

Reply:
and yeah, the purpose of this method is to not create a content facet when using the UI to create a host.  In that case the UI sends `:content_facet_attriabutes => {:lifecycle_environment_id => nil, :content_view_id => nil}`  when nothing is selected. In this case we do not want to create a content_facet